src/Dfe.PlanTech.Application/Content/MappedEntity.cs

```diff
@@ -202,7 +192,7 @@ private void UpdateProperties()
             var currentValue = property.GetValue(ExistingEntity);
 
             // Don't update the existing property if the values are the same
-            if (newValue?.Equals(currentValue) == true)
+            if ((newValue == null && currentValue == null) || (newValue != null && newValue.Equals(currentValue)))
```

Comment:
I'm curious, why change this? why not `newValue?.Equals(currentValue)` alone

Reply:
It kept failing on comparisons that _were_ equal for some reason; no idea why. This fixed it though. I've reverted the change now though as it seems to work as intended now, but still no clue why :D